docs/project/tokenomics.mdx

```diff
@@ -1,80 +1,135 @@
+
+#### 5% - Liquidity Bootstrap:
+will be used to bootstrap and support $WXM’s liquidity.
+
+#### 3% Airdrop" to beta participants
+3 million tokens will be used to reward stations that contributed data to the Network during the beta phase. This is not a marketing activity intended to create buzz and awareness, but a reward mechanism for the stations that participated and formed the Network before a reward mechanism was in place.
+
+Beta rewards will be allocated according to the number of days a station has provided valid data to the network during the testnet/beta period. The number of days contributed by each station during the beta period will be calculated using the latest rewards mechanism implemented at the time of the token launch and applied retrospectively to the data the stations provided during the beta period.
+ 
+Beta rewards will be distributed to beneficiaries gradually on top of the rewards they would normally receive for operating their stations, until all beta rewards allocated to them are distributed.
+
+For example, a station that provided data for 100 days before the token launch, will receive their beta rewards throughout the first 100 days after the token launch.
+
+This provides an additional incentive for the early Network adopters to remain in the network, maximising their station data value due to seniority and providing additional incentive for early members to actively participate in future Network governance.
+
+
+:::note
+To understand better the beta rewards you can read the related blog post <a href="https://blog.weatherxm.com/the-wxm-beta-rewards-program-fe2cbb9778b0">here</a>
+:::
+
+
+Not all tokens will be issued on token launch. More specifically:
+- 52M tokens will be  issued linearly, 14,246 per day, and will be  distributed daily to weather station owners.
+- 30M tokens will be issued daily using a 10-10-80% distribution in the first 3 years and will be distributed daily to WeatherXM AG and its Investors.
+- 3M tokens will be distributed retrospectively to station owners that participated in the beta phase of the Network (“Airdrop”).
+- A one-off issuance of 10M tokens will take place on token launch. These tokens will be used to support and fund the ecosystem development and the DAO formation.
+- A one-off issuance of 5M tokens will take place on token launch. These tokens will be used to provide market liquidity.
+
+
+## Architecture
+
+
+As more logic is transferred into smart contracts, a need for grouping of functionality and modularity emerges. That is why multiple smart contracts are now created, each with its own logic and design patterns:
+
+- WeatherXM
+- WeatherXMData
+- WeatherStationXM
+- RewardPool
+- BurnPool
+- PriceFeedConsumer
+
+Diagrams illustrating the interaction among the smart contracts, storage slots, classes, etc can be found <a href="https://github.com/WeatherXM/smart-contracts/tree/develop/docs/diagrams">diagrams</a>.

```

Comment:
Replace `diagrams` hyperlinked with `here` or added parentheses to it?

Reply:
done